tests/llmobs/llmobs_cassettes/tests.llmobs.test_llmobs_ragas_answer_relevancy_evaluator.emits_traces_and_evaluations_on_exit.yaml

```diff
@@ -0,0 +1,586 @@
+interactions:
```

Comment:
Are these cassettes also recording the evaluation payload submission to LLMObs? Can we simplify the tests to have just LLMObs writer assertions and a couple general end-to-end payload tests instead of having complex cassettes for every eval metric runner?